packages/react-next/src/components/Slider/Slider.test.tsx

```diff
@@ -119,30 +167,51 @@ describe('Slider', () => {
 
   it('formats the value when a format function is passed', () => {
     const value = 10;
-    const valueFormat = (val: any) => `${val}%`;
+    const valueFormat = (val: number) => `${val}%`;
     wrapper = mount(<Slider value={value} min={0} max={100} showValue={true} valueFormat={valueFormat} />);
 
     expect(wrapper.find('label.ms-Label.ms-Slider-value').text()).toEqual(valueFormat(value));
   });
 
-  it('calls onChanged after keyboard event', () => {
+  it('updates value correctly when down and up are pressed', () => {
+    const slider = React.createRef<ISlider>();
+    const onChange = jest.fn();
+
+    wrapper = mount(
+      <Slider label="slider" componentRef={slider} defaultValue={12} min={0} max={100} onChange={onChange} />,
+    );
+    const sliderSlideBox = wrapper.find('.ms-Slider-slideBox');
+
+    sliderSlideBox.simulate('keydown', { which: KeyCodes.down });
+    sliderSlideBox.simulate('keydown', { which: KeyCodes.down });
+    sliderSlideBox.simulate('keydown', { which: KeyCodes.down });
+    sliderSlideBox.simulate('keydown', { which: KeyCodes.up });
+    sliderSlideBox.simulate('keydown', { which: KeyCodes.down });
+
+    expect(slider.current?.value).toEqual(9);
+
+    // onChanged should only be called after a delay
+    expect(onChange).toHaveBeenCalledTimes(5);
+  });
+
+  it('correctly delays calling onChanged', () => {
     jest.useFakeTimers();
+    const slider = React.createRef<ISlider>();
     const onChanged = jest.fn();
 
-    const container = document.createElement('div');
-    document.body.appendChild(container);
+    wrapper = mount(
+      <Slider label="slider" componentRef={slider} defaultValue={12} min={0} max={100} onChanged={onChanged} />,
+    );
 
-    ReactDOM.render(<Slider label="slider" defaultValue={12} min={0} max={100} onChanged={onChanged} />, container);
-    const sliderSlideBox = container.querySelector('.ms-Slider-slideBox') as HTMLElement;
-
-    ReactTestUtils.Simulate.keyDown(sliderSlideBox, { which: KeyCodes.down });
-    ReactTestUtils.Simulate.keyDown(sliderSlideBox, { which: KeyCodes.down });
-    ReactTestUtils.Simulate.keyDown(sliderSlideBox, { which: KeyCodes.down });
-    ReactTestUtils.Simulate.keyDown(sliderSlideBox, { which: KeyCodes.up });
-    ReactTestUtils.Simulate.keyDown(sliderSlideBox, { which: KeyCodes.down });
+    const sliderSlideBox = wrapper.find('.ms-Slider-slideBox');
 
-    expect(sliderSlideBox.getAttribute('aria-valuenow')).toEqual('9');
+    sliderSlideBox.simulate('keydown', { which: KeyCodes.down });
+    sliderSlideBox.simulate('keydown', { which: KeyCodes.down });
+    sliderSlideBox.simulate('keydown', { which: KeyCodes.down });
+    sliderSlideBox.simulate('keydown', { which: KeyCodes.up });
+    sliderSlideBox.simulate('keydown', { which: KeyCodes.down });
 
+    expect(slider.current?.value).toEqual(9);
     // onChanged should only be called after a delay
```

Comment:
```suggestion
    // onChange should only be called after a delay
```

Reply:
This one is actually correct! onChanged is the prop that is called after a delay.